src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
```

Comment:
I think you should place it in a main block, but no sure
```suggestion
  <body class="page">
  <main class="main">
  ...code
  <main>
```